tests/desktop_web/test_performance_issues.py

```diff
@@ -0,0 +1,19 @@
+import time
+import pytest
+import sentry_sdk
+
+def test_nplusone(desktop_web_driver, endpoints, random, batch_size, sleep_length):
+    sentry_sdk.set_tag("pytestName", "test_nplusone")
+
+    for endpoint in endpoints['react_endpoints']:
+        endpoint_organization = endpoint + "/nplusone"
+        sentry_sdk.set_tag("endpoint", endpoint_organization)
+
+        url = endpoint_organization
+
+        for i in range(batch_size):
```

Comment:
[POSSIBLY REQUESTING CHANGE]

Not part of this PR, but

```
        for i in range(batch_size):
```
will generate batch_size * 20 hits to `flask`, so average of 200 requests. `/nplusone` right now only sleeps but still it's a lot of threads/processes. This could become a problem and require us to scale up flask GAE service.

on the other hand we want performance issues to show up high up-ish in the issue stream...

I'd suggest running this with BATCH_SIZE=40 (2x current max batch size) and seeing if anything abnormal happens and if it does change this to:
```
for i in range(max(1, batch_size//2)):
```

Reply:
understood. Making GH issue with this in it

Reply:
https://github.com/sentry-demos/application-monitoring/issues/186